CHANGELOG.md

```diff
@@ -12,6 +12,15 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Added the ability to set `PriorityClass` through `ScanSetting`. All the scanner pods
   will be launched using the `PriorityClass` if set. This is an optional attribute, the
   scan pods will be launched without `PriorityClass` if not set.
+- The `ScanSetting` Custom Resource now allows to override the default
+  CPU and memory limits of scanner pods through the `scanLimits` attribute.
+  Please refer to the [Kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/)
+  for syntax reference.
+  In order to set the memory limits of the operator itself, modify the
+  `Subscription` object, if installed through OLM or the operator deployment
+  itself otherwise. Increasing the memory limit for the operator or the scanner
+  might be needed in case the default limits are not sufficient and either the operator
+  or the scanner pods are killed by the OOM killer.
```

Comment:
Yes - this is better. What's likely to cause the increased memory usage? The number of nodes I'm scanning or is the the number of rules in the profile, etc.?

Reply:
We saw it primarily when users were able to deploy as watching all namespaces.

Reply:
In which case it was a lot of namespace and that required watch cache for huge number of configmaps, deployments, etc.. If we have a lot of nodes, or rules, those are singular objects so the cache is not in as much danger.

Reply:
Yes, I think also the nist profiles are quite memory intensive when scanning nodes because the number of RHCOS checks is high and some traverse the whole filesystem (or a large subset of).

Reply:
btw when we asked the SCAP team what's the memory usage they see in the RHEL world their answer was IIRC "anywhere from a few hundred megs to ~2GB". They did some memory improvements since, but still..

Reply:
Ok - so if I deploy the operator to watch all namespaces, and I have a lot of namespaces, I'd be interested in this.

Reply:
yes, the important thing being that if you install through the console (web ui) the default is to watch all namespaces.